### PR TITLE
Added MediaTek

### PR DESCRIPTION
**Pull request by ykodwani01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -480,6 +480,9 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Sigma Computing](https://boards.greenhouse.io/sigmacomputing/jobs/5371312003) | San Francisco, CA | Software Engineering Intern (Summer 2023) |
 | [Bandwidth](https://boards.greenhouse.io/bandwidth/jobs/4618588) | Rochester, NY | Software Engineering Intern (Summer 2023) |
 | [UKG](https://careers.ukg.com/careers/JobDetail/Software-Engineering-Internship-Spring-Summer-2023/41891) | Multiple | Software Engineering Internship – Spring & Summer 2023 (No Sponsorship) |
+| [Saras](https://www.linkedin.com/jobs/view/3302532590/?refId=dplI4qIFG8awj05GlZG4hQ%3D%3D&trackingId=4mXysicjcf3g7y%2FnXBYKuQ%3D%3D) | Hyderabad, Telangana, India, Hybrid | Software Engineer - Intern , Saras Analytics |
```
Comment (ewang2002) [inline README.md]: This repository is only for internships/co-ops in the United States, Canada or for Remote positions. Hybrid still implies that you need to be in that general location. Please remove.

File: README.md [hunk 2]
```diff
@@ -480,6 +480,9 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Sigma Computing](https://boards.greenhouse.io/sigmacomputing/jobs/5371312003) | San Francisco, CA | Software Engineering Intern (Summer 2023) |
 | [Bandwidth](https://boards.greenhouse.io/bandwidth/jobs/4618588) | Rochester, NY | Software Engineering Intern (Summer 2023) |
 | [UKG](https://careers.ukg.com/careers/JobDetail/Software-Engineering-Internship-Spring-Summer-2023/41891) | Multiple | Software Engineering Internship – Spring & Summer 2023 (No Sponsorship) |
+| [Saras](https://www.linkedin.com/jobs/view/3302532590/?refId=dplI4qIFG8awj05GlZG4hQ%3D%3D&trackingId=4mXysicjcf3g7y%2FnXBYKuQ%3D%3D) | Hyderabad, Telangana, India, Hybrid | Software Engineer - Intern , Saras Analytics | 
+| [MediaTek](https://www.linkedin.com/jobs/view/3324061814/?refId=3fvAnAfxQ1X2kZkN%2FLMhbw%3D%3D&trackingId=1p9gO%2Fn3s3eadD3LauTdag%3D%3D) | San Jose, CA | AI Architecture and Algorithm Intern |
```
Comment (ewang2002) [inline README.md]: Instead of a LinkedIn page, can you provide a link that goes _directly_ to the company's job page?

**Review by ewang2002 — CHANGES_REQUESTED** [review]
Hey, thanks for contributing! There are some issues with your changes that you will need to resolve beforehand. I just fixed some formatting issues.

File: README.md (hunk 2 above)
Comment (ykodwani01) [reply README.md]: > Instead of a LinkedIn page, can you provide a link that goes _directly_ to the company's job page?

Sure!

**Review by ewang2002 — APPROVED** [review]
Looks great! Thank you! :)

**Pull request merged**